src/Bicep.Core/PrettyPrintV2/SyntaxLayouts.Operators.cs

```diff
@@ -77,13 +78,27 @@ private Document Bracket(SyntaxBase openSyntax, Func<IEnumerable<Document>> item
                 .TrimNewlines()
                 .CollapseNewlines(onHardLine: this.ForceBreak)
                 .Concat(danglingComments)
-                .SeparateBy(separator);
+                .SeparateBy(separator)
+                .ToImmutableArray();
```

Comment:
FYI I think Anthony found that ImmutableArray/etc had a performance penalty vs Array/etc.  If this is performance sensitive might not want to use it for a private var.

Reply:
Good point. Changed it to ToArray.